packages/theme/components/MyAccount/ProfileUpdateForm.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div v-if="isAuthenticated && user">
+    <SfTable>
+      <SfTableRow>
+        <SfTableData>Email</SfTableData>
+        <SfTableData class="right-column">{{ userGetters.getEmailAddress(user) }}</SfTableData>
+      </SfTableRow>
+      <SfTableRow>
+        <SfTableData>Store Credits</SfTableData>
+        <SfTableData class="right-column">{{ userGetters.getStoreCredits(user) }}</SfTableData>
+      </SfTableRow>
+      <SfTableRow>
+        <SfTableData>{{ addresses && addresses[0] ? `Shipping to ${userShippingGetters.getFirstName(addresses[0])} ${userShippingGetters.getLastName(addresses[0])}` : 'Shipping' }}</SfTableData>
```

Comment:
Maybe create a computed method for `const mainAddress = computed(() => addresses[0])` and use it here?